src/pages/trade-types/cfds/_mind-when-trading.tsx

```diff
@@ -50,7 +50,7 @@ const MindWhenTrading = () => {
                     )}
                 </Text>
             </SmallContainer>
-            <Show.Desktop max_width={'bp680'}>
+            <Desktop breakpoint={size.bp680}>
```

Comment:
Can you modify the `Desktop` and `Mobile` component to accept both `numbers` or a key of `size` from `themes/device` ?